lib/utils/gchelper.h

```diff
@@ -28,7 +28,21 @@
 
 #include <stdint.h>
 
-uintptr_t gc_helper_get_sp(void);
-uintptr_t gc_helper_get_regs_and_sp(uintptr_t *regs);
+#if MICROPY_GCREGS_SETJMP
+#include <setjmp.h>
+typedef jmp_buf regs_t;
```

Comment:
Since this is now a public typedef, maybe the type name needs to be more specific, like `gc_helper_regs_t`?